client/css/overlays/players.css

```diff
@@ -1,23 +1,68 @@
+#playerOverlay h1{
+  margin-bottom: 0;
+}
+.instructionalText{
+  display: block;
+  font-size: 16px;
+  content: 'From here you can edit the name other players see as well as player colors.';
+  text-align: center;
+  margin-bottom: 1.5rem;
+}
+
 #playerList {
   overflow: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
 }
 
-.myPlayerEntry .playerName {
-  font-weight: bold;
+#playerList > div{
+  order: 2;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+  gap: 0 0.5rem;
+  padding: 0.25rem 0;
+  font-size: 0.5em;
+  line-height: 2rem;
+  font-size: 1rem;
+  width: 15rem;
 }
-
-.inactivePlayerEntry {
-  opacity: 0.5;
+#playerList > div input{
+  font-size: 1rem;
+  height: 2rem;
+  line-height: 2rem;
+}
+.inactivePlayerLine{
+  display: block;
+  height: 1px;
+  width: 100%;
+  order: 3;
+  overflow: hidden;
+  margin-top: -1px;
 }
-
 .teamColor {
-  border: none;
-  padding: 0;
-  margin: 0;
-  width: 20px;
-  height: 20px;
+  -webkit-appearance: none;
+  border-radius: 100%;
   display: inline-block;
-  border-radius: 20px;
+  margin: 0;
+  padding: 0;
+  width: 2rem;
+  height: 2rem;
+}
+
+.teamColor::-webkit-color-swatch-wrapper {
+  padding: 0;
+}
+
+.teamColor::-webkit-color-swatch {
+  border-radius: 100%;
+  border: 1px solid rgba(255, 255, 255, 0.25);
+  /* background properties broken up to avoid erasing the selected colors */
+  background-image: url(/i/icons-white/user-solid-svg.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 1rem;
 }
```

Comment:
Webkit does not work on firefox

Reply:
The appropriate -moz equivalents were added, which I didn't knew were things. Usually it's a struggle getting around all of Apple's ornamentation. I look forward to a day where we get rid of vendor prefixes.